Fixes for Web, Commands and Others

Incompleteusern opened the pull request:
The goal of this pr is to fix any issues with client commands, especially pertaining to `/wynntils reload`.

CringeAura commented:
Can someone explain how a **two-word change in a display text** causes a build to fail? (referring to commit #7)

Incompleteusern committed:
web content changes and territory command changes

change web content loading to be determined by caller

Incompleteusern committed:
component changes

Incompleteusern committed:
spotless

Incompleteusern committed:
fix crash

Incompleteusern committed:
add safety to apiurls accesses, fix territory not being dispatched

Also fix prompt for wynncraftServer

Incompleteusern committed:
spotless

Incompleteusern committed:
more fixes

Incompleteusern commented:
List of Fixes:
- Don't reset Wynntils Account (breaks things)
- A few checks to prevent crashes
- Fix `/wynntils help` having `reloadapi` still and being formatted incorrectly
- Fix Wynn Join Button not enabling resource packs by default
- Fix redirects being handled incorrectly by `ClientCommandManager`
- Fix `ItemsRecievedEvent` code being deleted
- Fix debug warning by mixins for unobfuscated code

Changes:
- Changed `TerritoryCommand` to only load territories when demanded
- Made it so that WebManager's `tryToLoadSomething` methods don't check whether the resource is already loaded, let call site decide
- Cleaned and changed component code

kristofbolyai commented on common/src/main/java/com/wynntils/commands/TerritoryCommand.java:
```diff
@@ -50,20 +57,24 @@ private MutableComponent helpComponent() {
+            context.getSource()
+                    .sendFailure(
+                            new TextComponent(
+                                            "Can't access territory " + "\"" + territoryArg + "\"")
```
Perhaps change this to some more descriptive text?

Incompleteusern committed:
add more descriptive comment and move compass manager

Incompleteusern committed:
fix item guesses and also add remap = false
